### ISSUE TITLE
HeadshotsAlwaysKill doesn't work

### ISSUE THREAD

**Issue by ezra-guzman** [paulov-t/SIT.Core#232]
**Describe the bug**
The bug is straightforward: headshots do not always kill despite HeadshotsAlwaysKill being set to true, regardless of helmet or not

**To Reproduce**
1. Load into a raid with a low damage weapon, like the Five-seveN, which will not kill an enemy in one shot in the base game
2. Shoot an enemy in the head
3. See that enemy doesn't die with the first shot

**Expected behavior**
If the `HeadshotsAlwaysKill` config is set to true, a headshot should always kill an enemy

**Screenshots**
N/A

**SIT.Core Version:**
SIT.Core.1.5.8572.14210

**SPT-Aki Version:**
3.5.8

**Logs**
Please provide the following log file(s) link here (upload into pastebin,etc)
https://u.pcloud.link/publink/show?code=VZ4HbUVZ8vVpSoz5DVfMzdCcXtunUh9UnQ5k

**Additional context**

**Comment by xjy646** [comment]
It is what the game should do? like if you use 4.6x30 AP SX ammo(damage 34) you can not kill an PMC in one shot. Helmet can reduce the damage. Some bosses even have more hp in head.

**Comment by ezra-guzman** [comment]
> It is what the game should do? like if you use 4.6x30 AP SX ammo(damage 34) you can not kill an PMC in one shot. Helmet can reduce the damage. Some bosses even have more hp in head.

I hear you. The point though, is that there is a config option for SIT which is enabled by default which makes it so that unarmored headshots kill automatically. See the original post for details

**Comment by xjy646** [comment]
In fact I think it is actually a game feature, in BSG version, you can encounter SCAVs tanking your shot with their head, wwearing no helmet (

**Comment by ezra-guzman** [comment]
> In fact I think it is actually a game feature, in BSG version, you can encounter SCAVs tanking your shot with their head, wwearing no helmet (

You are totally right, in the vanilla game many types of bullets do not kill with a single headshot. 

However the point of this post is that paulov has included a SIT config option which changes this so that, if someone is not wearing a helmet, headshots always kill. That config option is currently broken, which is the point of this post

**Comment by KoviUbi62** [comment]
I agree, it's not working how it should. Even when i headshot a bot in it's face where there's no armor it just doesn't die and the ammunition doesn't matter since I used M62 and .366TMK EKO. This also happened to me when I was the one getting shot 7 times in the face with buckshot(I did not wear faceshield and still survived).

**Comment by AngryOwl123** [comment]
Can confirm. I could shoot a scav with a .338 Lapua round and they will absolutely tank it. Any headshots will still take 2 to kill anyone whether PMC or scav.
